libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -446,7 +446,7 @@ func (m *Manager) Set(container *configs.Config) error {
 }
 
 func getUnitName(c *configs.Cgroup) string {
-	return fmt.Sprintf("%s-%s.scope", c.Parent, c.Name)
+	return fmt.Sprintf("%s-%s.scope", c.ScopePrefix, c.Name)
```

Comment:
I don't think "-xyz.scope" is valid. Pretty sure I saw systemd fail that name once :)